src/lib/content.d.ts

```diff
@@ -227,6 +227,10 @@ interface VideoVimeoBlockElement {
     height: number;
     width: number;
     caption: string;
+    html: string;
```

Comment:
I think `html` is rendundant here and can be removed

Reply:
good catch